learn/cookbooks/running_production.md

```diff
@@ -64,32 +68,44 @@ Meilisearch is finally installed and ready to use. To make it accessible from ev
 
 ```bash
 # Move the Meilisearch binary to your system binaries
-mv ./meilisearch /usr/bin/
+mv ./meilisearch /usr/local/bin/
 ```
 
-You can now start using Meilisearch! In your terminal, run the following command to launch meilisearch.
+## Step 2: Create user
+
+It is not a good idea run meilisearch as `root`. You should create own system user for meilisearch:
 
 ```bash
-meilisearch
+useradd -d /var/lib/meilisearch -b /bin/false -m -r meilisearch
```

Comment:
```suggestion
useradd -d /var/lib/meilisearch -b /bin/false -m -r meilisearch
usermod -aG sudo meilisearch
```

Reply:
I don't see why would meilisearch should have sudo rights. The reason for separate user is that it is unprivileged.

Reply:
Indeed it is not necessary at all to run Meilisearch as a service in this scenario.

I was considering any other additional config that you may be doing in the server, but it can always be done through `root` if it is not disabled. 

So you're probably right we may not want to include adding privileges to the meilisearch user int he guide 👌🏽

Reply:
Since indiscriminate `sudo` can be dangerous, I would prefer this example to only include the last line if it is strictly necessary for the majority of ours users. We don't want people to inadvertently weaken server security because they copypasted one of our code samples.